scripts/bootstrap_cluster.sh

```diff
@@ -119,10 +133,22 @@ install_argocd_helm_chart() {
     --set controller.extraArgs[0]='--application-namespaces="*"'
 }
 
+save_istio_values() {
```

Comment:
we are only doing it for tfy-istio-ingress? what about other charts?

Reply:
removed these